docs/sources/panels/field-configuration-options.md

```diff
@@ -83,19 +79,19 @@ Field overrides allow you to change the settings for one field (column in tables
 1. Continue to add overrides to this field by clicking **Add override property**, or you can click **Add override** and select a different field to add overrides to.
 1. When finished, click **Save** to save all panel edits to the dashboard.
 
-## Filter options
+## Override matchers
 
-This section explains all available filter options for field overrides. They are listed in alphabetical order.
+This section explains all available matchers for field overrides. They are listed in alphabetical order.
 
-### Filter field by name
+### Match field by name
```

Comment:
Please change these headings back. The original headings align more closely to the UI text.

Reply:
I did update the UI as well, as using the Filter word was a bit confusing. But think I have a better naming than match.

Add override dropdown:
* Fields matching name
* Fields matching regex
* Fields with type

Reply:
Works for me.